# HG changeset patch
# User John W. Eaton <jwe@octave.org>
# Date 1357960359 18000
#      Fri Jan 11 22:12:39 2013 -0500
# Branch stable
# Node ID e457139e7d8fa9116074a71c11947c4afe5e4c4b
# Parent  59128350e0119eb327b529ca226ac50d7074b668
* lo-cutils.h: Include sys/types.h.

diff --git a/liboctave/lo-cutils.h b/liboctave/lo-cutils.h
--- a/liboctave/lo-cutils.h
+++ b/liboctave/lo-cutils.h
@@ -18,16 +18,18 @@ You should have received a copy of the G
 along with Octave; see the file COPYING.  If not, see
 <http://www.gnu.org/licenses/>.
 
 */
 
 #if !defined (octave_liboctave_cutils_h)
 #define octave_liboctave_cutils_h 1
 
+#include "sys/types.h"
+
 #ifdef HAVE_LOADLIBRARY_API
 #define WIN32_LEAN_AND_MEAN
 #include <windows.h>
 #endif
 
 #ifdef __cplusplus
 extern "C" {
 #endif
